catalog/java/io/material/catalog/datepicker/DatePickerMainDemoFragment.java

```diff
@@ -163,13 +166,30 @@ public View onCreateDemoView(
             builder.setCalendarConstraints(constraintsBuilder.build());
             MaterialDatePicker<?> picker = builder.build();
             addSnackBarListeners(picker);
-            picker.show(getChildFragmentManager(), picker.toString());
+            picker.show(getChildFragmentManager(), "MaterialDatePicker");
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
+              view.post(() -> {
+                picker.requireDialog().getWindow().setNavigationBarContrastEnforced(
+                    !new WindowPreferencesManager(requireContext()).isEdgeToEdgeEnabled());
+              });
+            }
           } catch (IllegalArgumentException e) {
             snackbar.setText(e.getMessage());
             snackbar.show();
           }
         });
 
+    Fragment fragment = getChildFragmentManager().findFragmentByTag("MaterialDatePicker");
+    if (fragment instanceof MaterialDatePicker<?>) {
+      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
+        view.post(() -> {
```

Comment:
Move the whole view.pose(....) part to a private method to avoid duplicated codes?

Reply:
Thank you. I created navigationBarContrast() method.